### PR TITLE
[String] add test case for wordwrap method

### PR DESCRIPTION
**Pull request by mgamaltech**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| License       | MIT

Just test case for ```wordwrap``` method

### CODE REVIEW

File: src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php [hunk 1]
```diff
@@ -55,6 +55,17 @@ public static function provideBytesAt(): array
         ];
     }
 
+    public function testWordwrap()
+    {
+        $instance = static::createFromString('Lorem Ipsum');
+        $actual = $instance->wordWrap(5, '-');
```
Comment (nicolas-grekas) [inline src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php]: wordwrap

**Review by nicolas-grekas — COMMENTED** [review]
I think this deserves a few more input/output.
The logic of wordwrap is more than just replacing a space by a dash :)

File: src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php (hunk 1 above)
Comment (mgamal92) [reply src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php]: OK i will add dataProvider and more assertion here

Comment (mgamal92) [reply src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php]: @Nicals  i've added more test cases

File: src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php [hunk 2]
```diff
@@ -55,6 +55,53 @@ public static function provideBytesAt(): array
         ];
     }
 
+    /**
+     * @dataProvider wordwrapProvider
+     */
+    public function testWordwrap($expected, $actual, $length, $break, $cut=false)
+    {
+        $instance = static::createFromString($actual);
+        $actual = $instance->wordWrap($length, $break, $cut);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php]: wordwrap (lowercase "w")

File: src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php [hunk 3]
```diff
@@ -55,6 +55,53 @@ public static function provideBytesAt(): array
         ];
     }
 
+    /**
+     * @dataProvider wordwrapProvider
+     */
+    public function testWordwrap($expected, $actual, $length, $break, $cut=false)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php]: cs typo, see fabbot report

File: src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php [hunk 4]
```diff
@@ -55,6 +55,53 @@ public static function provideBytesAt(): array
...
+                '-',
+                true
+            ],
+            [
+                'Lorem-Ipsum',
+                'Lorem Ipsum',
+                2,
+                '-',
+            ],
+
```
Comment (nicolas-grekas) [inline src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php]: extra line

File: src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php [hunk 5]
```diff
@@ -55,6 +55,53 @@ public static function provideBytesAt(): array
...
+            [
+                'Lor-em-Ips-um',
+                'Lorem Ipsum',
+                3,
+                '-',
+                true
+            ],
+
```
Comment (nicolas-grekas) [inline src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php]: extra line

File: src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php [hunk 6]
```diff
@@ -55,6 +55,53 @@ public static function provideBytesAt(): array
...
+            [
+                'L-o-r-e-m-I-p-s-u-m',
+                'Lorem Ipsum',
+                1,
+                '-',
+                true
+            ]
+        ];
+    }
+
```
Comment (nicolas-grekas) [inline src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php]: extra line

**Review by nicolas-grekas — COMMENTED** [review]
LGTM, just minor issues remaining (see fabbot report)

**Comment by mgamal92** [comment]
i've fixed these issues :)

**Review by nicolas-grekas — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @mgamal92.

**Pull request merged**